src/lib/y2network/widgets/interfaces_table.rb

```diff
@@ -110,7 +110,7 @@ def interface_item(interface)
           # first is (item) ID in table
           interface.name,
           # if user named the connection explicitly, it will have precendence
-          conn&.description || friendly_name(interface),
+          conn&.description.to_s.empty? ? friendly_name(conn, interface) : conn.description
```

Comment:
well, I think it would be nice to move all that to that friendly_name method and maybe call it display_description or something like this?

Reply:
yep, sounds reasonable to me, already tried to move it, in fact that conn parameter is a leftover of the attempt which needs to be fixed :), just was not sure about the method description and passing both parameters so decided to not so far